### PR TITLE
Update the game over message

### PR DESCRIPTION
**Pull request by itstalmeez**
Update the game over message so people know how to play again!

### CODE REVIEW

**Review by itstalmeez — COMMENTED** [review]
Update the game over message so people know how to play again!

**Review by leenazaheer734 — APPROVED** [review]
ok

**Review by leenazaheer734 — COMMENTED** [review]
i will mark the changes

**Comment by leenazaheer734** [comment]
i have reviewed

**Comment by mohfarukranabappy** [comment]
support

**Review by SarahSidAhmed — APPROVED** [review]

**Review by SarahSidAhmed — COMMENTED** [review]
Reviewed

**Review by simeenpathan — COMMENTED** [review]
This is technical onboarding by MLSA

File: index.html [hunk 1]
```diff
@@ -24,6 +24,7 @@
       background: #222;
       color: #ccc;
     }
+/*     So here we go */
   }
 </style>
```
Comment (simeenpathan) [inline index.html:30]:
```suggestion
<p>Hello World</p>
```

**Review by spamshot — APPROVED** [review]
ll

**Review by spamshot — COMMENTED** [review]
ll

**Comment by Mohith19** [comment]
How will this works!

**Comment by alirazahub** [comment]
great

**Review by alirazahub — COMMENTED** [review]
reviewed

**Review by alirazahub — APPROVED** [review]
reviewed

**Review by alirazahub — CHANGES_REQUESTED** [review]
reviewed

**Comment by Ivanovich0705** [comment]
reviewed

**Pull request closed without merging**